sdk/purview/azure-purview-sharing/swagger/README.md

```diff
@@ -0,0 +1,45 @@
+# Azure Purview for Python
+
+> see https://aka.ms/autorest
+
+### Setup
+
+Install Autorest v3
+
+```ps
+npm install -g autorest
+```
+
+### Generation
+
+```ps
+cd <swagger-folder>
+autorest
+```
+
+### Settings
+
+```yaml
+package-name: "@azure-rest/purview-sharing"
+title: Purview Sharing Client
+description: Purview Sharing Client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json
+output-folder: ../azure/purview/sharing
+namespace: azure.purview.sharing
+clear-output-folder: true
+no-namespace-folders: true
+python: true
+package-version: 1.0.0b1
+rest-level-client: true
+add-credential: true
+credential-scopes: "https://purview.azure.net/.default"
+generate-test: false
+generate-samples: true
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
Hi @yamanwahsheh Please follow https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-with-tools to generate the code again. Your `swagger/README.md` is outdated.

Reply:
@yamanwahsheh  This conversation is marked as resolved but the `swagger/README.md` is not updated. I help commit some change to it and regenerate the SDK code again with `autorest --version=3.9.2 --use=@autorest/python@6.4.3 --use=@autorest/modelerfour@4.25.0 README.md`. After the commit,  there may be some CI error, please follow https://github.com/Azure/azure-sdk-for-python/wiki/How-to-fix-CI to fix it

Reply:
@msyyc the swagger/readme.md was changed according to the provided link. I will re-run the command as stated here too and check for updates, and will keep an eye for the error.

Reply:
Just did, and re-generated the files, also re-ran the tests.